scripts/init

```diff
@@ -6,7 +6,15 @@ devenv() {
   . ${DEVENVROOT}/scripts/func.d/bash_utils
   . ${DEVENVROOT}/scripts/env.d/devenv_impl
 
-  if [[ "$1" == "use" ]]; then
+  if [[ "$1" == "" ]]; then
+    if [ -z "${DEVENVFLAVOR}" ] ; then
+      display "no active flavor: \$DEVENVFLAVOR not defined"
+    else
+      display "'${DEVENVFLAVOR}' is active: ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}"
+    fi
+    display ""
```

Comment:
In the second commit 882c52f4d3b2437bfa2609f3ca9f703164af86e1 I also include the help message when printing.